deepmd/pd/utils/stat.py

```
@@ -0,0 +1,604 @@
+
+
+def _compute_model_predict(
+    sampled: Union[Callable[[], list[dict]], list[dict]],
+    keys: list[str],
+    model_forward: Callable[..., paddle.Tensor],
+):
+    auto_batch_size = AutoBatchSize()
+    model_predict = {kk: [] for kk in keys}
+    for system in sampled:
+        nframes = system["coord"].shape[0]
+        coord, atype, box, natoms = (
```

Ruff

Local variable `natoms` is assigned to but never used

Remove assignment to unused variable `natoms`

(F841)